lab-brandon/ContainsDuplicates.java

```diff
@@ -0,0 +1,15 @@
+public class ContainsDuplicates {
+
+    public static boolean containsDuplicates(int[] arr) {
+        boolean isDuplicate = false;
+
+        for (int i = 0; i < arr.length; i++) {
```

Comment:
There is a way to do this without the nested `for` loops - but you can leave this as-is if you want, it's totally valid. :)